### DISCUSSION TITLE
Feature request: Sorting in search results

### DISCUSSION THREAD

**Discussion by cowgoesmoo69** [tidal-music/discussions#62] [Ideas & Feature requests]
It would be really handy to be able to sort search results. For example, a popular DJ magazine, Mixmag, has released loads of albums over the years, guest DJ mixes etc. (other examples are Global Underground, FABRICLIVE)

If I do a search in the desktop app for mixmag and then choose Albums I get a list of albums in no particular order. ![2024-04-27T13_59_06Z TIDAL COW-RTX](https://github.com/tidal-music/discussions/assets/17062157/959f90a3-2499-43aa-a561-3846a7930a79)

What would be great here are sorting options, e.g. chronologically, or by album artist (the album artist tends to be whoever is the guest DJ doing the mix).

**Comment by stefanmoro** [comment]
Hey @tomearp ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product feature, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).
